### PR TITLE
fix: [#9655] - Fix compilation errors under Angular 16

### PR DESCRIPTION
**Pull request by kekonn**
## Description 📝
This is a fix for issue #9655 .

## Fixes 🔄
**List highlighting major changes**
- Change `src/regions` to `../regions` in `linode/types.ts`
- Change `src/regions` to `../regions` in `account/types.ts`

### CODE REVIEW

File: packages/api-v4/src/account/types.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { APIWarning } from '../types';
-import { Region } from 'src/regions';
+import { Region } from '../regions';
```
Comment (bnussman-akamai) [inline packages/api-v4/src/account/types.ts:2]:
```suggestion
import type { Region } from '../regions';
```

File: packages/api-v4/src/linodes/types.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Region } from 'src/regions';
+import { Region } from '../regions';
```
Comment (bnussman-akamai) [inline packages/api-v4/src/linodes/types.ts:1]:
```suggestion
import type { Region } from '../regions';
```

**Review by bnussman-akamai — APPROVED** [review]
Great change! We should always be using relative imports where it makes sense.

**Review by coliu-akamai — APPROVED** [review]
Thanks for catching @kekonn!

@bnussman-akamai would we need a changeset for this PR?

**Comment by bnussman-akamai** [comment]
Yes, good call @coliu-akamai

**Review by jdamore-linode — APPROVED** [review]
Also calling out that the e2es have passed -- the tests will kick off again and automatically fail when the changeset gets pushed, but I don't think re-running the tests will be necessary at that point

**Pull request merged**